__tests__/app.test.js

```diff
@@ -68,5 +68,24 @@ describe('Get all comments by id', () => {
             });
         });
     });
-});
-});
+
+    test('should return fail and return 404', () => {
```

Comment:
we should describe with a bit more detail why we are expecting this to fail an example test description would be: "404: when passed a valid but non-existent article_id"